quick-start/Ch-QuickStart.rst

```diff
@@ -36,6 +36,87 @@ Finally, verify that the EdgeX containers have all been started::
 
 .. image:: EdgeX_GettingStartedUsrActiveContainers.png
 
+The following table captured the Default Service Ports. This table provides the default ports used by each of the EdgeX micro services (per its default configuration and the EdgeX provided docker-compose files).  These default ports are also used in the EdgeX provided service routes defined in the Kong API Gateway for access control.
+
+
+.. csv-table:: Port Number
+   :file: DefaultPort.csv
+   :widths: 60, 40
+   :header-rows: 1
+
+.. csv-table:: The contents of my xlsx sheet exported to mytable.csv
+   :widths: 60 40
+   :header: "Header 1", "Header 2", "Header 3"
+   :file: DefaultPort.csv
```

Comment:
I don't see this file in the repository.

Reply:
use list-table and it works, the csv-table method did not work